server/lib/configuration.js

```diff
@@ -370,6 +370,20 @@ var conf = module.exports = convict({
     env: 'FXA_OAUTH_URL',
     format: 'url'
   },
+  openid_configuration: {
+    response_types_supported: ['code', 'token'],
```

Comment:
Yea, I see the lint complaining. I initially wrote them in the same order as was in Google's doc. Easy to rearrange.